tests/posix/common/src/pthread.c

```diff
@@ -574,33 +512,7 @@ static void *create_thread1(void *p1)
 	return NULL;
 }
 
-ZTEST(posix_apis, test_pthread_create_negative)
```

Comment:
hey, sorry I have a hard time following, are these deleted or moved? the commit message suggets that they are moved but the commit only seems to delete

Reply:
They're moved. Probably will be more apparent when I squash the last two commits into one.

Reply:
Fixed!